### DISCUSSION TITLE
Show "ride time" in the stats of flat rides

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#18850] [Feature requests]
Would be nice to be able to see the ride duration here:
![Geoffrey Gardens 2022-12-14 20-50-13](https://user-images.githubusercontent.com/26573303/207778471-27b64bc2-d12d-4d4d-8d95-02e634199545.png)
The same way coasters and other tracked rides show it:
![Geoffrey Gardens 2022-12-14 20-50-13_mod](https://user-images.githubusercontent.com/26573303/207779757-0e472361-813f-4c41-93b7-085dc0012433.png)

It would be especially nice for flat rides that don't have visible operation *and* have variable ride duration, depending on settings, like the 3D Cinema here.

But ride duration would be a nice stat to see on *all* flat rides. 
(Where applicable, of course. It wouldn't make sense to show ride time for things like a Maze or a Spiral Slide that's set to 'unlimited rides per admission', of course, since in those cases ride duration depends on a peep's decisions.)
It would help you plan parks more efficiently and better estimate throughput. And on rides like the 3D Cinema, it would give you a more informed way to weigh the pros and cons of selecting which film to show. (One film gives higher stats than the other, but also has a longer runtime. How much longer? ... It's hard to tell, because the 3D Cinema doesn't report ride duration!) It would also be nice to see how setting the number of swings on a Pirate Ship or the number of rotations on a Merry-Go-Round affects ride duration ... without having to get out an actual physical stopwatch!

**Comment by spacek531** [comment]
Flat ride cycle time is hardcoded in weird ways. Having the invisible reviewer determine cycle time would likely be easier than refactoring flat rides so ride cycle time has a consistent prediction method.

**Reply by ocalhoun6** [reply]
Yeah, for sure. But the invisible reviewer is already visiting flat rides anyway, right? Because excitement/intensity/nausea ratings are still being calculated. So hopefully it would be a pretty simple change to have the ride reviewer also report ride duration?
